Stream_support/test/Stream_support/CMakeLists.txt

```diff
@@ -7,13 +7,27 @@ project( Stream_support_Tests )
 cmake_minimum_required(VERSION 3.1)
 
 find_package(CGAL QUIET)
-
 if ( CGAL_FOUND )
+  find_path(3MF_INCLUDE_DIR
+    NAMES Model/COM/NMR_DLLInterfaces.h
+    DOC "Path to lib3MF headers"
+    )
+  find_library(3MF_LIBRARIES NAMES 3MF DOC "Path to the lib3MF library")
 
   # create a target per cppfile
   file(GLOB cppfiles RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
   foreach(cppfile ${cppfiles})
-    create_single_source_cgal_program( "${cppfile}" )
+    if ( "${cppfile}" STREQUAL "test_3mf_to_sm.cpp" )
+      if(3MF_LIBRARIES AND 3MF_INCLUDE_DIR)
```

Comment:
same question as for the demo

Reply:
Which is ?

Reply:
Basically, why two different find.

Reply:
Because there is no find_package() for lib3mf, so we need to do it manually, which requires a find for the lib and a file for the headers.